drivers/coreaudio/audio_driver_coreaudio.h

```diff
@@ -120,6 +120,6 @@ class AudioDriverCoreAudio : public AudioDriver {
 	~AudioDriverCoreAudio();
 };
 
-#endif
-
 #endif // AUDIO_DRIVER_COREAUDIO_H
+
+#endif // COREAUDIO_ENABLED
```

Comment:
This should really be inside the header guard and not outside IMO, i.e. the ifdef should be moved inside instead